lib/realtime/vehicle.ex

```diff
@@ -188,6 +190,7 @@ defmodule Realtime.Vehicle do
       longitude: VehiclePosition.longitude(vehicle_position),
       direction_id: direction_id,
       route_id: route_id,
+      route_pattern_id: trip && trip.route_pattern_id,
```

Comment:
**suggestion (non-blocking):** since the other fields like `headsign` that pull data from `trip` do the lookup logic above, it may be easier to read if this is instead grouped at the beginning of the function with them.